logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/slow/SlowSyncManager.kt

```diff
@@ -103,7 +103,9 @@ internal class SlowSyncManager(
                 logger.i("Last SlowSync was performed on '$lastTimeSlowSyncWasPerformed'")
                 val nextSlowSyncDateTime = lastTimeSlowSyncWasPerformed + MIN_TIME_BETWEEN_SLOW_SYNCS
                 logger.i("Next SlowSync should be performed on '$nextSlowSyncDateTime'")
-                currentTime > nextSlowSyncDateTime
+                val lastVersion = slowSyncRepository.getSlowSyncVersion()
+                logger.i("Last saved SlowSync version is $lastVersion, current is $CURRENT_VERSION")
+                currentTime > nextSlowSyncDateTime || CURRENT_VERSION > lastVersion
```

Comment:
👏

Reply:
We should add some documentation though, as to when is it advisable to bump the `CURRENT_VERSION`. Maybe also add it to the `FeatureConfigEventReceiver` and/or the `SyncFeatureConfigsUseCase`